docker-compose.override.yml

```diff
@@ -23,13 +23,24 @@ services:
       - "5102:80"
       - "5581:5001"
 
+  courseregistration.api:
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ASPNETCORE_URLS=http://0.0.0.0:80
+      - ConnectionStrings=Server=sql.data;Database=OpenCodeFoundation.CourseRegistrationDb;User Id=sa;Password=Pass@word
+    ports:
+      - "5103:80"
+      - "5582:5001"
+
   webstatus:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=http://0.0.0.0:80
       - ConnectionStrings=Server=sql.data;Database=OpenCodeFoundation.WebStatusDb;User Id=sa;Password=Pass@word
       - HealthChecksUI__HealthChecks__0__Name=Enrolling HTTP Check
       - HealthChecksUI__HealthChecks__0__Uri=http://enrolling.api/hc
+      - HealthChecksUI__HealthChecks__0__Name=CourseRegistration HTTP Check
```

Comment:
This is an array of configurations. So your need to increase the number. for example -

```
HealthChecksUI__HealthChecks__1__Name=CourseRegistration HTTP Check
...
```

Reply:
Updated. Please check in new commit.